src/components/common/HelperText.tsx

```diff
@@ -0,0 +1,27 @@
+import {
+    FormErrorIcon,
+    FormErrorMessage,
+    FormHelperText
+} from '@chakra-ui/react';
+
+interface HelperTextProps {
+    hasError?: boolean;
```

Comment:
how can everything be optional?

Reply:
When `<HelperText />` is used to show error message, `hasError` and `errorMsg` is passed. And when normal helper message is shown, `msg` is passed. It is possible to make `hasError` required, but in that case, `hasError={false}` has to be passed everytime the component is used to show helper message

Reply:
Note to self: use types to make sure the right prop is passed in both `hasError={false}` and `hasError={true}` case

Reply:
Changed type so that atleast one of `msg` or `errorMsg` should always be passed.